src/event.ts

```diff
@@ -392,7 +412,15 @@ export interface WatchRunEnd extends InfoBase {
 }
 
 /**
- * A service started running.
+ * A service process started running.
+ */
+export interface ServiceProcessStarted extends InfoBase {
+  detail: 'service-process-started';
+}
+
+/**
+ * A service started running and if it has a readyWhen condition,
+ * that condition is met.
  */
 export interface ServiceStarted extends InfoBase {
   detail: 'service-started';
```

Comment:
How about renaming this old event to `service-ready`, to distinguish it more clearly from `service-process-started`? The "ready" terminology is what we use in the configuration file and docs to mean that the ready condition, if there is one, has been satisfied (e.g. "Server started" was logged).

Reply:
+1, done. Also logged `[serviceName] Service ready` in the default logger instead of `[serviceName] Service started`